lib/tasks/efile.rake

```diff
@@ -23,4 +23,8 @@ namespace :efile do
     puts("End: Found #{EfileSubmission.in_state(:failed).count} failed submissions")
     exit
   end
+
+  task send_survey_notifications: :environment do
```

Comment:
Hmm I don't _love_ that we are putting this task in this file. I think I would prefer that this file is just for efile tasks that we use regularly. For example, `poll_and_get_acknowledgments` we run in cron every 10 minutes and I assume `retry_failed_submissions` was used semi regularly with CTC failed submissions. I think I would prefer it was in a separate rake task and that all the logic in the service is just placed in that rake task instead of getting called from a service. And then eventually one day we just remove that rake task.

Reply:
I've moved all the code into `lib/tasks/send_survey_notifications.rake`